src/main/java/teammates/ui/pagedata/InstructorFeedbackEditPageData.java

```diff
@@ -176,9 +176,20 @@ private boolean isVisibilitySetToAnonymousToRecipientAndInstructors(FeedbackQues
     }
 
     private boolean isVisibilitySetToAnonymousToRecipientVisibleToInstructors(FeedbackQuestionAttributes question) {
-        boolean responsesVisibleOnlyToRecipientAndInstructors = question.showResponsesTo.size() == 2
-                && question.showResponsesTo.contains(FeedbackParticipantType.INSTRUCTORS)
-                && question.showResponsesTo.contains(FeedbackParticipantType.RECEIVER);
+        boolean responsesVisibleOnlyToRecipientAndInstructors;
+
+        if (question.questionType.equals(FeedbackQuestionType.CONTRIB)) {
```

Comment:
I am not sure if we want to treat this as a special case for `AnonymousToRecipientVisibleToInstructors`.  @junhaoyap What do you think? Should we create another name for this case?